charts/scalardb/Chart.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: scalardb
+description: A Helm chart of Scalar DB
+
+type: application
+version: 2.0.0
+appVersion: 3.1.0
+deprecated: false
+icon: https://scalar-labs.com/wp-content/themes/scalar/assets/img/logo_scalar.svg
+keywords:
+- scalardb
+- scalardb-server
+home: https://scalar-labs.com/
+sources:
+  - https://github.com/scalar-labs/scalardb/tree/master/server
```

```suggestion
  - https://github.com/scalar-labs/scalardb
```
I don't think we should add a branch here.
ref. https://github.com/cockroachdb/helm-charts/blob/master/cockroachdb/Chart.yaml